.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Rust",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "runArgs": ["--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],
+
+  "settings": {
+    "lldb.executable": "/usr/bin/lldb",
+    // VS Code don't watch files under ./target
+    "files.watcherExclude": {
+      "**/target/**": true
+    }
+  },
+
+  "extensions": [
+    "matklad.rust-analyzer",
+    "bungcip.better-toml",
+    "vadimcn.vscode-lldb",
+    "mutantdino.resourcemonitor",
+    "denoland.vscode-deno"
```

Comment:
Do you really want this one installed?  It becomes a little problematic of having a version built in the container? Also, it doesn't do so well with internal code, I have `"deno.enable": false` for my vscode for Deno.  Maybe consider at least setting up some default `deno.` settings in the `devcontainer.json`?

Reply:
Removed.